src/nvim/lua/vim.lua

```diff
@@ -309,7 +309,9 @@ setmetatable(vim, {
 })
 
 -- An easier alias for commands.
-vim.cmd = vim.api.nvim_command
+vim.cmd = function(command)
+  vim.api.nvim_exec(command, false)
```

Comment:
One thing that just came to my mind.. Wouldn't it be better to use `return vim.api.nvim_exec(...)` here instead? I feel like that would help the compiler to remove one call stack.

@bfredl @tjdevries

Reply:
`nvim_command` returns nothing but `nvim_exec` returns an empty string . Wouldn't that cause a deference ?

Reply:
What deference? mhinz is thinking of tail call optimization. The empty string gets referenced/unreferenced regardless.

Reply:
```lua
if vim.cmd('')
```

> mhinz is thinking of tail call optimization.

I don't know what that is . If you guys think it's needee I'll add it :)

Reply:
Yeah, please add `return` and I'll merge it right away. ;]

Reply:
Alright added . Though did any one vote on you poll ?